frontend/src/lib/components/Badges/Badge.svelte

```diff
@@ -1,23 +1,12 @@
 <script lang="ts">
-	import { Badge } from '.';
-
 	export let type = '';
```

Comment:
I should have mentioned this in the last PR, but could we specify the valid strings here? in this case it's really just going to be injected into the class so you can use anything, but just for usability for a dev can we specify the valid strings here?